be/src/util/aes_util.cpp

```diff
@@ -160,10 +160,10 @@ int AesUtil::decrypt(AesMode mode, const unsigned char* encrypt, uint32_t encryp
         return AES_BAD_DATA;
```

Comment:
aes_ctx will leak?
Better to create aes_ctx after this if clause

Reply:
Yes, it's possible of leaking. Will correct it and commit again.